python/experiments/generate-box-tests-reports.main.kts

```diff
@@ -26,19 +26,24 @@ import javax.xml.parsers.DocumentBuilderFactory
 import kotlin.text.Charsets.UTF_8
 
 fun pathFromNamedArgument(argumentName: String): Path? =
+    stringFromNamedArgument(argumentName)?.let { Paths.get(it) }
+
+fun stringFromNamedArgument(argumentName: String): String? =
     args.firstOrNull { it.startsWith("--$argumentName=") }
-        ?.let { Paths.get(it.split("=")[1]) }
+        ?.let { it.split("=")[1] }
 
+val testTask: String = stringFromNamedArgument("test-task")
+    ?: throw IllegalArgumentException("'test-task' parameter is required!")
```

Comment:
Could we leave default value as "pythonTest" here? Would be handy. We can add a warning that it's used when it's not specified.

```suggestion
val testTask: String = stringFromNamedArgument("test-task")
    ?: "pythonTest".also { println("'test-task' parameter is missing, using default 'pythonTest'") }
```